Frontend/CO.CDP.OrganisationApp/Pages/ContactUs.cshtml

```diff
@@ -0,0 +1,117 @@
+@page "/contact-us"
+@model ContactUsModel
+
+@{
+    var nameHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Name)).HasInnerHtml;
+    var emailAddressHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.EmailAddress)).HasInnerHtml;
+    var organisationNameHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.OrganisationName)).HasInnerHtml;
+    var messageHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Message)).HasInnerHtml;
+}
+
+<main class="govuk-main-wrapper" id="main-content">
+    @if (Model.MessageSent == true)
+    {
+        <div class="govuk-grid-row">
+            <div id="standard-left-col" class="standard-col govuk-grid-column-two-thirds">
+                <div class="standard-col-content">
+                    <div class="govuk-panel govuk-panel--confirmation break-word">
+                        <h1 class="govuk-panel__title">
+                            Message sent
+                        </h1>
+                    </div>
+                    <br />
+                    <h2 class="govuk-heading-m">What happens next</h2>
+                    <br>
+                    <p class="govuk-body">Our support team will contact you by email, Monday to Friday from 9am to 5pm, excluding bank holidays.</p>
+                </div>
+            </div>
+        </div>
```

Comment:
Markup needs some things removing - looks like some extra classes copied and pasted from somewhere else.

```suggestion
        <div class="govuk-grid-row">
            <div class="govuk-grid-column-two-thirds">
                  <div class="govuk-panel govuk-panel--confirmation">
                      <h1 class="govuk-panel__title">
                          Message sent
                      </h1>
                  </div>
                  <h2 class="govuk-heading-m">What happens next</h2>
                  <p class="govuk-body">Our support team will contact you by email, Monday to Friday from 9am to 5pm, excluding bank holidays.</p>
            </div>
        </div>
```

But I also wonder, could we use the FlashMessage pattern for this instead? (Not super fussed about this, the panel is fine if it looks ok and I guess this is to be consistent with FTS?)